src/Steeltoe.All.sln

```diff
@@ -702,8 +702,8 @@ Global
 		{A9F88613-375A-4CE8-9B34-D88799850570}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{A9F88613-375A-4CE8-9B34-D88799850570}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{A9F88613-375A-4CE8-9B34-D88799850570}.Release|Any CPU.Build.0 = Release|Any CPU
-		{D5572167-00EB-48FA-8CC3-0394A399C289}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{D5572167-00EB-48FA-8CC3-0394A399C289}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{D5572167-00EB-48FA-8CC3-0394A399C289}.Debug|Any CPU.ActiveCfg = Release|Any CPU
+		{D5572167-00EB-48FA-8CC3-0394A399C289}.Debug|Any CPU.Build.0 = Release|Any CPU
```

Comment:
```suggestion
		{D5572167-00EB-48FA-8CC3-0394A399C289}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
You might have had this right before... I usually just have this project unloaded so it never affected me

Reply:
So this wasnt working for me as expected - so I reverted this. And good idea to just unload.